wgsl/index.bs

```diff
@@ -146,8 +146,8 @@ A shader comprises:
 When executing a shader stage, the implementation:
 * Binds [=resources=] to variables in the shader's [=resource interface of a shader|resource interface=],
     making the contents of those resources available to the shader during execution.
-* Allocates storage for other [=module scope|module-scope=] variables,
-    and populates that storage with the specified initial values.
+* Allocates memory for other [=module scope|module-scope=] variables,
```

Comment:
I'm confused, based on your PR description should these be `memory storage` instead? It still seems inconsistently used in this secton.

Reply:
Oh, you're right.  I think the commit message fell behind the sequence of edits I had made privately.

By the way, the thinking is:
"memory storage" by itself seems redundant, and I prefer just "memory".
After all, what is memory but a place to store stuff.

But "memory storage space" seems better than "memory space".

This appears once in:
> Also, each invocation has its own independent memory storage space in the form

Reply:
I updated the commit message